### ISSUE TITLE
wtmi build warning: get_ref_clk()

### ISSUE THREAD

**Issue by afaerber** [MarvellEmbeddedProcessors/A3700-utils-marvell#3]
Using openSUSE Tumbleweed cross-gcc-arm-none-gcc7's arm-none-eabi-gcc 7.3.1 I get:
```
[  132s]   CC    clock.c
[  132s] clock.c: In function 'set_tbg_clock':
[  132s] clock.c:397:6: warning: implicit declaration of function 'get_ref_clk'; did you mean 'get_ddr_clock'? [-Wimplicit-function-declaration]
[  132s]   if (get_ref_clk() == 40) {
[  132s]       ^~~~~~~~~~~
[  132s]       get_ddr_clock
```

```
ddr/wtmi_ddr/clock.c:   if (get_ref_clk() == 40) {
...
wtmi/clock.h:u32 get_ref_clk(void);
```

Should `ddr/wtmi_ddr/clock.c` or `ddr/wtmi_ddr/clock.h` do `#include "../../wtmi/clock.h"`?

**Comment by afaerber** [comment]
Using wtmi/clock.h leads to even more warnings (struct redefinition). Copying the single declaration into clock.c worked okay.

**Comment by kostapr** [comment]
Does this issue still exist?

**Comment by pali** [comment]
Looks like that this issue is already fixed in A3700_utils-armada-18.12-fixed branch. At leas I'm not seeing it.
